### PR TITLE
Removing @private from aliasMethod

### LINKED ISSUE #12238: aliasMethod marked as private
Ember.aliasMethod was marked as private without notice.  Can we remove the `@private` or is it in fact now private?

https://github.com/emberjs/ember.js/blob/v2.0.1/packages/ember-metal/lib/mixin.js#L772

### PR DESCRIPTION
**Pull request by jasonmit**
Fixes #12238

### CODE REVIEW

File: packages/ember-metal/lib/mixin.js [hunk 1]
```diff
@@ -702,7 +702,6 @@ Alias.prototype = new Descriptor();
   @method aliasMethod
   @for Ember
   @param {String} methodName name of the method to alias
-  @private
```
Comment (rwjblue) [inline packages/ember-metal/lib/mixin.js]: Every documentation block must include @public or @private (to ensure it is super clear what is and is not private).  Our test suite enforces this.

This @private was likely added while cleaning up all blocks. 

Can you add @public here instead?

**Comment by jasonmit** [comment]
@rwjblue all done and squashed.  Thanks

**Comment by rwjblue** [comment]
Thank you!

**Comment by jasonmit** [comment]
No, thank you.  You're awesome :+1:

**Pull request merged**